docs/react-native/background.mdx

```diff
@@ -54,3 +54,34 @@ You need to modify `AndroidManifest.xml` file and add below service:
 
   </TabItem>
 </Tabs>
+
+## Usage
+
+<Tabs groupId="app-type">
+    <TabItem value="ios" label="iOS">
+      This feature is unavailable on iOS.
+    </TabItem>
+
+  <TabItem value="android" label="Android">
+
+You can use `useForegroundService` hook to handle how foreground service behaves on Android.
+
+:::warning[Permissions]
+
+If you want to use `enableCamera` or `enableMicrophone`, user must first grant permission for this resource. `useForegroundService` will check if permission is granted and only then allow to start a service.
+
+:::
+
+```tsx
+useForegroundService({
+  channelId: "io.fishjam.example.fishjamchat.foregroundservice.channel",
+  channelName: "Fishjam Chat Notifications",
+  notificationTitle: "Your video call is ongoing",
+  notificationContent: "Tap to return to the call.",
+  enableCamera: isCameraOn,
+  enableMicrophone: isMicrophoneOn,
+});
+```
+
+  </TabItem>
+</Tabs>
```

Comment:
**suggestion (non-blocking):** Let's maybe swap those two, so a working example is visible first.